### PR TITLE
docs: Fixed resource alicloud_log_dashboard document error.

### PR DESCRIPTION
**Pull request by MrWolong**

### CODE REVIEW

File: website/docs/r/log_dashboard.html.markdown [hunk 1]
```diff
@@ -73,27 +74,27 @@ EOF
...
+* `id` - The resource ID in terraform of Log Dashboard. It formats as `<project_name>:<dashboard_name>`.
 
 ## Import
 
-Log Dashboard can be imported using the id or name, e.g.
+Log Dashboard can be imported using the id, e.g.
 
 ```shell
-$ terraform import alicloud_log_dashboard.example tf-project:tf-logstore:tf-dashboard
+$ terraform import alicloud_log_dashboard.example <project_name>:<dashboard_name>
```
Comment (MrWolong) [inline website/docs/r/log_dashboard.html.markdown:99]: 文档资源id是错的，正确的应该是个二级id：`<project_name>:<dashboard_name>`
`d.SetId(fmt.Sprintf("%s%s%s", d.Get("project_name").(string), COLON_SEPARATED, d.Get("dashboard_name").(string)))`

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Comment by ChenHanZhang** [comment]
文档中JSON要用<<EOF 或者 jsonencode

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Pull request merged**